flake8-requirements.txt

```diff
@@ -1,27 +1,27 @@
-flake8~=3.8.3
+flake8==3.8.3
```

Comment:
My fix that lets people be able to use flake8 on more devices should be in 3.8.4, so we should keep an eye out.